qcloud_cos/cos_auth.py

```diff
@@ -3,24 +3,31 @@
 
 import random
 import time
-import urllib
 import hmac
 import hashlib
 import binascii
 import base64
 
+try:
+    from urllib import quote  # Python 2.X
+except ImportError:
+    from urllib.parse import quote  # Python 3+
+
+from .helper import smart_str, smart_bytes
```

Comment:
I am not sure whether this importation is worked on Python3. Python3 supports `absolute import` only. It would be better to update to ` from qcloud_cos.helper import smart_str, smart_bytes`.